tools/istio-iptables/pkg/dependencies/implementation_linux.go

```diff
@@ -244,11 +253,8 @@ func (r *RealDependencies) executeXTablesWithOutput(cmd constants.IptablesCmd, i
 		}
 	} else {
 		if needLock {
-			// We want the lock. Wait up to 30s for it.
-			args = append(args, "--wait=30")
 			c = exec.Command(cmdBin, args...)
 			log.Debugf("running with lock")
-			mode = "with wait lock"
```

Comment:
should keep this, it will say 'without lock' now even though it does have a lock

Reply:
Yep, not sure why I dropped that, ty